File: Java/Program/Product.java
import java.util.List;
import java.util.Scanner;

class Product{
    //atribut
    String id;
    String name;
    String brand;
    Double price;

    //contructors
    Product(){

    }

    //contructor w/ parameter
    Product(String id, String name, String brand, Double price){
        this.id = id;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }

    //bikin get sama set
    //get id
    String getId(){
        return this.id;
    }

    //set id
    void setId(String id){
        this.id = id;
    }

    String getName(){
        return this.name;
    }

    //set name
    void setName(String name){
        this.name = name;
    }

    String getBrand(){
        return this.brand;
    }

    //set name
    void setBrand(String brand){
        this.brand = brand;
    }

    Double getPrice(){
        return this.price;
    }

    //set name
    void setPrice(Double price){
        this.price = price;
    }
}

File: PHP/Program/product.php
<?php
class Product {
    //atribut yang dilindungi dan hanya dapat diakses oleh kelas turunan
    protected $idProduct; //untuk id produk
    protected $name; //untuk nama produk
    protected $brand; //untuk merek produk
    protected $price; //untuk harga produk

    //contructors
    public function __construct($idProduct, $name, $brand, $price) {
        $this->idProduct = $idProduct; //inisialisasi id produk
        $this->name = $name; //inisialisasi nama produk
        $this->brand = $brand; //inisialisasi merek produk
        $this->price = $price; //inisialisasi harga produk
    }

    //get id produk
    public function getIdProduct() {
        return $this->idProduct;
    }

    //get nama produk
    public function getName() {
        return $this->name;
    }

    //get merek produk
    public function getBrand() {
        return $this->brand;
    }

    //get harga produk
    public function getPrice() {
        return $this->price;
    }
}
?>
